subsys/net/l2/openthread/Kconfig

```diff
@@ -316,7 +316,6 @@ config OPENTHREAD_MAC_SOFTWARE_CSMA_BACKOFF_ENABLE
 
 config OPENTHREAD_CRYPTO_PSA
 	bool "ARM PSA crypto API"
-	depends on BUILD_WITH_TFM
```

Comment:
True, TF-M is not the only provider of the PSA APIs.
If mbedtls is enabled with PSA support you can still have PSA APis

Maybe update to:
depends on MBEDTLS_PSA_CRYPTO_C || BUILD_WTH_TFM?

Reply:
Sounds good